src/controllers/project.js

```diff
@@ -63,7 +63,7 @@ const getCommentByID = async (req, res, next) => {
       throw error;
     }
     //
-    const comment = await project.comments.find({ 'comment._id': commentid });
+    const comment = await project.find({ 'comments._id': commentid });
```

Comment:
the model is Project btw so here it should be Project instead of lower case project

Reply:
but I am searching inside the specific project that I got from its id a few lines before :thinking_face:

Reply:
nooo, the find method works on mongodb models your project here is a json object which does not have a find method your database operations should be done on mongodb model, if you test your code you will get what i mean because find is not a function on the project you found instead you should do Project.find which is ModelName.find()

Reply:
btw if you your point is to combine both finding a project then finding a comment/review inside that project then you need to aggregate or other syntax i guess since the comments, reviews are embedded something similar the following `const comment = await Project.find({
      $and: [{ _id: id }, { 'comments._id': commentid }],
    });`

Reply:
actually, this is what I was having in mind
![image](https://user-images.githubusercontent.com/26306192/151971786-cac80fc7-185b-4c5c-bc83-0cf5d215acbd.png)